corehq/apps/data_interfaces/tasks.py

```diff
@@ -113,7 +113,7 @@ def check_data_migration_in_progress(domain, last_migration_check_time):
     '{domain}',
     timeout=36 * 60 * 60,
     max_retries=0,
-    queue='background_queue',
+    queue=settings.CELERY_CASE_UPDATE_RULE_QUEUE,
```

Comment:
i agree this should be sent to a different queue, but going forward we're going to be hard-coding queue names in the task definition instead of allowing to be overridden in localsettings, so we can just set this directly to `case_rule_queue` or something